### ISSUE TITLE
Levels and types in switch menu

### ISSUE THREAD

**Issue by nasserkessas** [Pokemon-PythonRed/Pokemon-PythonRed#122]
**Feature Description**:

- I think the switch menu should include the Pokemon's level and type as well as the name and HP

Current
```
Which Pokémon should you switch to?

[1] - Charmander (18/18)
[2] - Rattata (15/15)
[3] - Pidgey (16/16)
[4] - Weedle (14/14)
[e] - Back
```

Proposed
```
Which Pokémon should you switch to?

[1] - Charmander (18/18) - Level 7 (FIRE)
[2] - Rattata (15/15) - Level 4 (NORMAL)
[3] - Pidgey (16/16) - Level 5 (NORMAL)
[4] - Weedle (14/14) - Level 3 (BUG)
[e] - Back
```

**Comment by TurnipGuy30** [comment]
I'll do this shortly.

**Comment by TurnipGuy30** [comment]
Solved with b3fbc94b23e7fd27bb60f7709f71475cdc0ff4ea
